pkg/kp/rc.go

```diff
@@ -0,0 +1,14 @@
+package kp
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationControllerStore interface {
+	Create(manifest pods.Manifest, nodeSelector, podSelector labels.Selector) (string, error)
```

Comment:
> You need to know how to label the pod as well as select it.

I actually made it so that it's the label set (so the replication controller knows how to label the pod), and I expect the replication controller to make the selector based on that label set.